tests/test_legacy_external_class.py

```diff
@@ -0,0 +1,63 @@
+from nose.tools import assert_true, assert_list_equal, raises
+from numpy.testing import assert_almost_equal
+import datajoint as dj
+from . import schema_legacy_external as modu
+
+
+def test_heading():
+    heading = modu.Simple().heading
+    assert_true('item' in heading)
+    assert_true(heading['item'].is_external)
+
+
+def test_insert_and_fetch():
+    original_list = [1, 3, 8]
+    modu.Simple().insert1(dict(simple=1, item=original_list))
```

Comment:
Same as above; should the `()` be removed throughout this method?

Reply:
This is inside legacy tests.  We support both syntaxes.